cli/providers/aws/translate.go

```diff
@@ -70,9 +70,12 @@ func loadASG(t *gocf.Template, s *service.Service, e *environment.Environment, a
 	}
 	asgResource := &gocf.Resource{Properties: asg, UpdatePolicy: updatePolicy}
 	t.Resources["ASG"] = asgResource
+	if s.Size == "" {
```

Comment:
what do you think about having a per provider value, and if that's empty, then fallback to this?